src/factories/workflow-specific/PIM_WorkflowFactory.sol

```diff
@@ -0,0 +1,187 @@
+            fundingManagerConfig,
+            authorizerConfig,
+            paymentProcessorConfig,
+            moduleConfigs
+        );
+
+        // get bonding curve (= funding manager) address
+        address fundingManager = address(orchestrator.fundingManager());
+        issuanceToken.setMinter(fundingManager, true);
+
+        // revoke minter role from factory
+        issuanceToken.setMinter(address(this), false);
+
+        // if renounced token flag is set, renounce ownership over token, else transfer ownership to initial admin
+        if (PIMConfig.isRenouncedIssuanceToken) {
+            _transferTokenOwnership(issuanceToken, address(0));
+        } else {
+            _transferTokenOwnership(
+                issuanceToken, PIMConfig.issuanceTokenParams.initialAdmin
+            );
+        }
+
+        // if renounced workflow flag is set, renounce admin rights over workflow, else transfer admin rights to initial admin
+        if (PIMConfig.isRenouncedWorkflow) {
+            _transferWorkflowAdminRights(orchestrator, address(0));
+        } else {
+            _transferWorkflowAdminRights(
+                orchestrator, PIMConfig.issuanceTokenParams.initialAdmin
+            );
+        }
+
+        _manageInitialCollateral(
+            fundingManager,
+            PIMConfig.collateralToken,
+            PIMConfig.bcProperties.initialCollateralSupply
+        );
+
+        emit IPIM_WorkflowFactory.PIMWorkflowCreated(
+            fundingManager,
+            address(issuanceToken),
+            msg.sender,
+            PIMConfig.recipient,
+            PIMConfig.isRenouncedIssuanceToken,
+            PIMConfig.isRenouncedWorkflow
+        );
+
+        return (orchestrator, issuanceToken);
+    }
+
+    //--------------------------------------------------------------------------
+    // onlyOwner Functions
+
+    /// @inheritdoc IPIM_WorkflowFactory
+    function setFee(uint newFee) external onlyOwner {
+        fee = newFee;
+        emit IPIM_WorkflowFactory.FeeSet(newFee);
+    }
+
+    /// @inheritdoc IPIM_WorkflowFactory
+    function withdrawFee(IERC20 token, address to) external onlyOwner {
+        token.transfer(to, token.balanceOf(address(this)));
+    }
+
+    //--------------------------------------------------------------------------
+    // Internal Functions
+
+    function _manageInitialCollateral(
+        address fundingManager,
+        address collateralToken,
+        uint initialCollateralSupply
+    ) internal {
+        IERC20(collateralToken).transferFrom(
+            msg.sender, fundingManager, initialCollateralSupply
+        );
+
+        if (fee > 0) {
+            uint feeAmount = _calculateFee(initialCollateralSupply);
+            IERC20(collateralToken).transferFrom(
+                msg.sender, address(this), feeAmount
+            );
+        }
+    }
+
+    function _transferTokenOwnership(
+        ERC20Issuance_v1 issuanceToken,
+        address newAdmin
+    ) private {
```

Comment:
Did you intentionally make this one `private` while `_manageInitialCollateral` is `internal`?